### PR TITLE
GitHub/actions unit test

### PR DESCRIPTION
**Pull request by 0xLeif**
Source: https://github.com/webbertakken/unity-actions/blob/master/.github/workflows/main.yml

- [x] Fix Scripts have compiler errors.

### CODE REVIEW

**Comment by 0xLeif** [comment]
@ambid17 @FuhrmannMaciej 

I tried running the project locally and I was getting a lot of errors... Screen Shot 2020-04-18 at 1 22 43 PM

**Comment by 0xLeif** [comment]
CD: [Build Target](https://docs.unity3d.com/ScriptReference/BuildTarget.html)

File: .github/workflows/CD.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+          projectPath: BunnyHopMaster
+          unityVersion: 2019.2.11f1
+
+      # Build
+      - name: Build project
+        uses: webbertakken/unity-builder@v0.10
+        with:
+          projectPath: BunnyHopMaster
+          unityVersion: 2019.2.11f1
+
```
Comment (FuhrmannMaciej) [inline .github/workflows/CD.yml:42]: I might be wrong but maybe there are errors because our unity version for bunnyhop is 2019.3.0f3.

Comment (0xLeif) [reply .github/workflows/CD.yml:42]: I'll try that version! Thanks <3

Comment (0xLeif) [reply .github/workflows/CD.yml]: oof ...

https://github.com/marketplace/actions/unity-test-runner#unityversion
> unityVersion
> 
> Version of Unity to use for testing the project.
> 
> required: false default: 2019.2.1f11

Comment (0xLeif) [reply .github/workflows/CD.yml]: Found: https://hub.docker.com/layers/gableroux/unity3d/2019.3.9f1/images/sha256-ca190e0394accbf3bcf2cd15db08cfd1296e96f6e2888e33b31e4ba7b0f92833?context=explore

[From](https://hub.docker.com/r/gableroux/unity3d/tags)

**Pull request merged**